flink-end-to-end-tests/run-pre-commit-tests.sh

```diff
@@ -53,6 +53,30 @@ if [ $EXIT_CODE == 0 ]; then
     EXIT_CODE=$?
 fi
 
+if [ $EXIT_CODE == 0 ]; then
+    printf "\n==============================================================================\n"
+    printf "Running Resuming Savepoint (no parallelism change) end-to-end test\n"
+    printf "==============================================================================\n"
+    $END_TO_END_DIR/test-scripts/test_resume_savepoint.sh 2 2
+    EXIT_CODE=$?
+fi
```

Comment:
The changes in this file seems to be unintended, and should be removed.

Reply:
will change.